core/src/main/java/org/bitcoinj/core/ECKey.java

```diff
@@ -708,12 +708,7 @@ public static boolean verify(byte[] data, ECDSASignature signature, byte[] pub)
             return true;
 
         if (Secp256k1Context.isEnabled()) {
-            try {
```

Comment:
This seems like it opens up the possibility for malformed keys to throw an unhandled exception -- probably not what one would want. Difficult to see where the exception would be handled in the call chain, probably somewhere, so this wouldn't take down a node... but anyways.